docs/reference/manifest/appdomain.md

```diff
@@ -7,14 +7,16 @@ Specifies an additional domain that will be used to load pages in the add-in win
 ## Syntax
 
 ```XML
-<AppDomain>string </AppDomain>
+<AppDomain>string</AppDomain>
 ```
 
+> [!IMPORTANT]
+> The **AppDomain** must include the protocol e.g., `<AppDomain>https://myappdomain"<AppDomain>`.
+
 ## Contained in
 
 [AppDomains](appdomains.md)
 
 ## Remarks
 
-The  **AppDomains** and **AppDomain** elements are used to specify any additional domains other than the one specified in the SourceLocation element. For more information, see Office Add-ins XML manifest.
-
+The  **AppDomains** and **AppDomain** elements are used to specify any additional domains other than the one specified in the [SourceLocation element](sourcelocation.md). For more information, see [Office Add-ins XML manifest](/office/dev/add-ins/develop/add-in-manifests).
```

Comment:
Suggest rephrasing the start of this sentence -> "**AppDomain** elements should be used to specify..."